### PR TITLE
[W5.11][W13-A2] Aadyaa Maddi

### PR DESCRIPTION
**Pull request by amad-person**
Enhancement 2: 
Made the FindCommand case insensitive.

### CODE REVIEW

**Comment by william6364** [comment]
I think Contact.java should not have been included in this PR as it makes it unclean, should add it under a separate PR

**Comment by amad-person** [comment]
Right, thanks for pointing that out!

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -50,11 +50,30 @@ public CommandResult execute() {
...
+
+            if (!Collections.disjoint(wordsInNameIgnoreCase, keywordsIgnoreCase)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Returns list of strings after converting each string to lower case
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java:65]: `Converts the list of strings to lower case` is more concise, no?

**Comment by okkhoy** [comment]
@amad-person 
Some comments added. Please ack & close the PR after reading comments.

**Comment by amad-person** [comment]
Acknowledged, thanks!

**Pull request closed without merging**